digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/RailwayCrossingService.scala

```diff
@@ -19,6 +22,91 @@ class RailwayCrossingService(val roadLinkService: RoadLinkService) extends Point
     persistedAsset.copy(floating = floating)
   }
 
+  override def getByBoundingBox(user: User, bounds: BoundingRectangle): Seq[PersistedAsset] = {
```

Comment:
It seems like there is a lot of duplicated code we can have a method on the base class that has a function as a parameter, and then reuse it.